### PR TITLE
#99 Add Career journey and Education in Profile Page

### PR DESCRIPTION
**Pull request by karyna-kh**

### CODE REVIEW

**Review by AndriyCh1 — APPROVED** [review]

File: frontend/package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "react-bootstrap-icons": "1.7.2",
     "react-cropper": "2.1.8",
     "react-bootstrap-typeahead": "5.2.2",
+    "react-datepicker": "^4.6.0",
```
Comment (aleksandr-kuharenko) [inline frontend/package.json]:
```suggestion
    "react-datepicker": "4.6.0",
```

**Review by aleksandr-kuharenko — CHANGES_REQUESTED** [review]

File: frontend/src/common/types/http/http-options.type.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { ContentType, HttpMethod } from 'common/enums/enums';
 type HttpOptions = {
   contentType: typeof ContentType[keyof typeof ContentType];
   method: keyof typeof HttpMethod;
-  payload: BodyInit | null;
+  payload?: BodyInit | null;
```
Comment (nikita-remeslov) [inline frontend/src/common/types/http/http-options.type.ts]: not sure, but `null` is redundant

File: frontend/src/react-app-env.d.ts [hunk 3]
```diff
@@ -1,2 +1,3 @@
 /// <reference types="react-scripts" />
 declare module 'react-notifications';
+declare module 'react-datepicker';
```
Comment (nikita-remeslov) [inline frontend/src/react-app-env.d.ts:3]: I think we can use https://www.npmjs.com/package/@types/react-datepicker

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
Overall looks good, fix few comments and we can merge this PR

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**